log4j-core/src/test/java/org/apache/logging/log4j/core/LogEventTest.java

```diff
@@ -27,6 +27,7 @@
 import org.apache.logging.log4j.core.impl.Log4jLogEvent;
 import org.apache.logging.log4j.message.Message;
 import org.apache.logging.log4j.message.SimpleMessage;
+
```

Comment:
We don't require whitespace here. We don't disallow it either, but not sure what the point is.